upcloud/service/utils_test.go

```diff
@@ -53,8 +51,8 @@ func handleError(err error) {
 	}
 }
 
-// records the API interactions of the test.
-func record(t *testing.T, fixture string, f func(*testing.T, *recorder.Recorder, *Service)) {
+// records the API interactions of the test. Function provides both services to test cases so that old utility functions can be used to initialize environment.
+func recordWithContext(t *testing.T, fixture string, f func(context.Context, *testing.T, *recorder.Recorder, *Service)) {
```

Comment:
`WithContext` suffix could probably be removed as obsolete, same for other helper functions with that in their name

Reply:
Good point. Pushed changes,